File: hackpdm_odoo/hackpdm/custom_addons/custom_module/models/hp_property.py
from odoo import models, fields, api

class hp_property(models.Model):
    #base fields
    _name = 'hp.property'
    _description = 'hp property'
    _inherit = 'hp.common.model'

    #fields
    name = fields.Char(string='property name', required=True)
    prop_type = fields.Char(string='property type')
    active = fields.Boolean(string='active')

    #relational fields

File: HackPDM_CSharp/OdooClient/OdooModels/Models/HpProperty.cs
//using static System.Net.Mime.MediaTypeNames;

namespace HackPDM
{
    public class HpProperty : HpBaseModel<HpProperty>
    {
        public string name;
        public string prop_type;
        public bool active;

        public HpProperty() { }
        public HpProperty(
            string name,
            string prop_type = null,
            bool active = default)
        {
            this.name = name;
            this.prop_type = prop_type;
            this.active = active;
        }
		public override string ToString()
		{
			return name;
		}
	}
}
